examples/dht_sensor/dht_sensor.c

```diff
@@ -0,0 +1,39 @@
+/* 
+ *
+ * This sample code is in the public domain.
+ */
+#include "espressif/esp_common.h"
+#include "esp/uart.h"
+#include "FreeRTOS.h"
+#include "task.h"
+#include "dht.h"
+#include "esp8266.h"
+
+/* An example using the ubiquitous DHT** humidity sensors
+ * to print out a new temperature and humidity measurement
+ * every three seconds.
+ */
+int const gpio4 = 4;

```

Comment:
It is probably a good idea to make the GPIO used be easily configurable, but if you're going to do this, it should probably be named something more general (like maybe `dht_gpio`)..  Currently if somebody decides to change this then they'll end up with code that says (for example) `gpio4 = 5`, which is just all kinds of confusing :)